### PR TITLE
Switch builds to Node 22

### PR DESCRIPTION
**Pull request by greenscientist**
Use node 22 for CI and docker builds by default

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Build Node app
-FROM node:18-bullseye
+FROM node:22-bullseye
```
Comment (tahini) [inline Dockerfile:2]: Why bookworm in transition and bullseye in evolution?

**Review by tahini — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (greenscientist) [reply Dockerfile:2]: Because of trRouting/OSRM dependency in Transition, it's harder to upgrade.

Comment (greenscientist) [reply Dockerfile:2]: Oh, I guess we just did not update it

Comment (greenscientist) [reply Dockerfile:2]: I'll add a commit to update to bookworm

**Review by tahini — APPROVED** [review]

**Review by samuel-duhaime — APPROVED** [review]

**Comment by greenscientist** [comment]
I've also added the latest node in the lint step

**Pull request merged**